Nodejs/Setup/NodejsTools.vsmanproj

```diff
@@ -48,6 +48,7 @@
   <Target Name="Build" />
   <Target Name="Clean" />
   <Target Name="ReBuild" />
+  <Target Name="BuiltProjectOutputGroup" />
```

Comment:
This fixes a new build error I'm seeing on my dev machine, this doesn't change the output for this project.